packages/block-library/src/image/edit.native.js

```diff
@@ -390,7 +395,13 @@ class ImageEdit extends React.Component {
 						} }
 					</ImageSize>
 					{ ( ! RichText.isEmpty( caption ) > 0 || isSelected ) && (
-						<View style={ { padding: 12, flex: 1 } }>
+						<View
+							style={ { padding: 12, flex: 1 } }
+							accessible={ true }
+							accessibilityLabel={ __( 'Image caption' ) + __( '.' ) + ' ' + ( isEmpty( caption ) ? __( 'Empty' ) : caption ) }
+							accessibilityHint={ __( 'Double tap to edit caption' ) }
+							accessibilityRole={ 'button' }
```

Comment:
I'm not sure if the hint is really necessary here. Is true that from the label, it's not obvious that this can be edited, but as is marked as a `button` it denotes that it's actionable. After the action the user gets in the text field on edit mode.

If we keep it, I'd add a `the` between `edit` and `caption` (Double tap to edit **the** caption). But better to ask someone English native to be sure :)